### PR TITLE
remove references to preview

### PR DESCRIPTION
**Pull request by sebastian-lerner**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Review by thomasr8 — APPROVED** [review]

**Comment by rosieyohannan** [comment]
@thomasr8 you need to merge PRs after approval, only docs team can merge now :-D

**Pull request merged**
